package.json

```diff
@@ -21,8 +21,8 @@
     "test:a11y": "echo 'TODO: Accessibility tests'",
     "test:nsp": "nsp check",
     "test:cucumber": "protractor test/config/cucumber.conf.js",
-    "test:smoke": "protractor test/config/smoke.conf.js",
-    "test:smokeDocker": "./bin/runSmokeTests.sh",
+    "test:smoke": "./bin/runSmokeTests.sh",
+    "test:smokeTest": "protractor test/config/smoke.conf.js",
     "test:functional": "echo 'TODO: Enable functional tests' || ./bin/runFunctionalTests.sh",
```

Comment:
You were doing so well on those commit messages.  Please correct the commit message for this.

I'm lost which needs to change ito in docker or no in docker.  But I'll trust you on thos.

Reply:
👍